learn/getting_started/installation.mdx

```diff
@@ -155,6 +166,106 @@ To deploy Meilisearch on a cloud service, follow one of our dedicated guides:
 - [Qovery](/learn/cookbooks/qovery)
 - [Railway](/learn/cookbooks/railway)
 
+### Installing older versions of Meilisearch
+
+<Tabs.Container labels={["cURL", "Docker", "Source", "Direct download"]}>
+
+<Tabs.Content label="cURL">
+
+View all available binaries under [GitHub assets](https://github.com/meilisearch/meilisearch/releases).
+
+```bash
+# Replace {meilisearch_version} and {meilisearch_os} with the specific version and OS you want to download
+# For example, if want to download v1.0 on macOS, replace {meilisearch_version} and {meilisearch_os} with v1.0 and meilisearch-macos-amd64 respectively
+curl -OL https://github.com/meilisearch/meilisearch/releases/download/{meilisearch_version}/{meilisearch_os}
+
+# Rename binary to meilisearch. Replace {meilisearch_os} with the name of the downloaded binary
+mv {meilisearch_os} meilisearch
+
+# Give the binary execute permission
+chmod +x meilisearch
+
+# Launch Meilisearch
+./meilisearch
+```
+
+</Tabs.Content>
+
+<Tabs.Content label="Docker">
+
+When using **Docker**, you can run [any available tag](https://hub.docker.com/r/getmeili/meilisearch/tags).<br />
+
+```bash
+# Fetch specific version of Meilisearch image from DockerHub. Replace vX.Y.Z with the version you want to use
+docker pull getmeili/meilisearch:vX.Y.Z
+
+# Launch Meilisearch in development mode with a master key
+docker run -it --rm \
+    -p 7700:7700 \
+    -e MEILI_ENV='development' \
+    -v $(pwd)/meili_data:/meili_data \
+    getmeili/meilisearch:vX.Y.Z
+# Use ${pwd} instead of $(pwd) in PowerShell
+```
+
+You can learn more about [using Meilisearch with Docker in our dedicated guide](/learn/cookbooks/docker).
+
+</Tabs.Content>
+
+<Tabs.Content label="Source">
+
+Meilisearch is written in `Rust`. To compile it, [install the Rust toolchain](https://www.rust-lang.org/tools/install).<br />
+
+If the Rust toolchain is already installed, clone the repository on your local system and change it to your working directory.
+
+```bash
+git clone https://github.com/meilisearch/meilisearch
+cd meilisearch
+```
+
+Choose the release you want to use. You can find the full list [here](https://github.com/meilisearch/meilisearch/releases).<br />
```

Comment:
Another `<br />`. Why do some paragraphs have them and other don't?

Reply:
The tags are added between two sentences to add a new line:
![Screenshot 2023-05-08 at 17 30 44](https://user-images.githubusercontent.com/90181761/236837311-63eea497-b3d9-4e56-a205-8a7d43e2bc4f.png)

According to Basement, we need `<br>` tags as just starting the sentence in a new line won't work in containers.

But it looks like it isn't working:
![Screenshot 2023-05-08 at 17 28 38](https://user-images.githubusercontent.com/90181761/236837048-3cfae51f-6595-4146-9f5d-bf56a13fd249.png)

>Why do some paragraphs have them and other don't?

It's only between two sentences. You won't find it if we have a code sample after a sentence or a last sentence:
![Screenshot 2023-05-08 at 17 38 22](https://user-images.githubusercontent.com/90181761/236838853-80d1d538-239c-4d0b-b046-80b7dec936b2.png)

Does that make sense?